packages/circuit-ui/components/Hamburger/Hamburger.tsx

```diff
@@ -166,6 +166,7 @@ export const Hamburger = ({
   ...props
 }: HamburgerProps): JSX.Element => {
   if (
+    process.env.UNSAFE_DISABLE_ACCESSIBILITY_ERRORS !== 'true' &&
     process.env.NODE_ENV !== 'production' &&
     process.env.NODE_ENV !== 'test' &&
     (!activeLabel || !inactiveLabel)
```

Comment:
Wondering if there's a way to abstract this away somewhere to avoid 3 duplicate lines in 30ish components

Reply:
I don't think there is a way without adding a function call in production everywhere 😕